Indicators/IndicatorExtensions.cs

```diff
@@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
             return second;
         }
 
+        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
```

Comment:
You got it!

On Sat, Apr 16, 2016 at 3:08 PM, Jared <notifications@github.com> wrote:

> In Indicators/IndicatorExtensions.cs
> <https://github.com/QuantConnect/Lean/pull/379#discussion_r59974857>:
>
> > @@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
> >              return second;
> >          }
> >
> > +        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
>
> Can you please add a XML comment here? It will help autocomplete users
> understand how to use it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/QuantConnect/Lean/pull/379/files/ccf1d4a664c7b27d07ff81210a939900b2542f0b#r59974857>
>



-- 

Best regards,

Mike Ellertson
Email: mdellertson@gmail.com